fenix/buildSrc/src/main/java/FenixDependencies.kt

```diff
@@ -88,6 +89,7 @@ object FenixDependencies {
     const val androidx_profileinstaller = "androidx.profileinstaller:profileinstaller:${FenixVersions.androidx_profileinstaller}"
     const val androidx_biometric = "androidx.biometric:biometric:${FenixVersions.androidx_biometric}"
     const val androidx_fragment = "androidx.fragment:fragment-ktx:${FenixVersions.androidx_fragment}"
+    const val androidx_activity_compose = "androidx.activity:activity-compose:${FenixVersions.androidx_activity_compose}"
```

Comment:
I think we should explicitly add `androidx.activity-ktx` dependency as well as that gets [pulled in with this](https://android.googlesource.com/platform/frameworks/support/+/refs/heads/androidx-main/activity/activity-compose/build.gradle#32) and we haven't explicitly declared that dependency in the project yet.

Reply:
@rahulsainani added the dependency for `androidx.activity-ktx`

Reply:
thanks!